sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokersTest.java

```diff
@@ -593,6 +595,62 @@ public void processElement(@SuppressWarnings("unused") ProcessContext c) {}
     invoker.invokeFinishBundle(null);
   }
 
+  @Test
+  public void testOnTimerHelloWord() throws Exception {
```

Comment:
Done. Notes going forward: This caught the fact that `OnTimerInvoker` was package private, thus invisible to subclasses.